packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -44,9 +33,19 @@ The `@customElement` decorator defines a custom element, equivalent to calling:
 customElements.define('my-element', MyElement);
 ```
 
-The `@property` decorator declares a reactive property.
+The `@property` decorator declares a [reactive property](/docs/v3/components/properties/).
+
+{% aside "info" %}
+
+Decorator versions and syntax.
+
+Lit decorators support multiple versions of the decorator specification – TypeScript experimental decorators and standard decorators – but to be used as standard decorators require a newer syntax of using the `accessor` keyword on decorated properties.
+
+We recommend that TypeScript users stick with experimental decorators for the moment, and as such, all of our TS code samples contain syntax that work in that mode, _without_ the `accessor` keyword.
+
+If you are using Babel to compile decorators, or are required to use `"experimentalDecorators": false` in your TypeScript config, you will need to add those. See [Standard decorators](#standard-decorators) for more information.
```

Comment:
I think this is a bit long for a note. I would mostly just direct the reader to the **Decorator versions** section:

```suggestion
Lit supports two different versions of the JavaScript decorators proposal – an early version supported by TypeScript that we refer to as _experimental decorators_ and a new and final version we refer to as _standard decorators_.
 
There are some small differences in usage between the two proposals (standard decorators sometimes require the `accessor` keyword). We use the experimental decorators conventions in our code samples because we recommend experimental decorators for production at the moment.
 
 See [Decorator versions](#decorator-versions) for more details.
```

Reply:
Nice! I definitely felt like this was long but wasn't sure how to make it more concise.

Reply:
The second paragraph with 3 line beginnings starting with "decorators" and line endings with "experimental" is awkward. 😂 

<img width="781" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/4c7c02bf-2389-4893-857d-212428edfcbb">

Reply:
yeah. Let's see...

```suggestion
Lit supports two different versions of the JavaScript decorators proposal – an early version supported by TypeScript that we refer to as _experimental decorators_ and a new and final version we refer to as _standard decorators_.
 
There are some small differences in usage between the two proposals (standard decorators sometimes require the `accessor` keyword). Our code samples use experimental decorators because we recommend them for production at the moment.
 
See [Decorator versions](#decorator-versions) for more details.
```

Reply:
Looking better..
<img width="800" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/af655b75-5dee-4b49-929b-26e10947a71e">

How about instead of "Our code samples use experimental decorators", "Our code samples are written to work with experimental decorators", or "Our code samples are written for experimental decorators" ? It's a subtle difference but I don't think "use" is right because they're all "using" the same hybrid decorators.

Reply:
I originally had "use experimental decorators conventions" but I dropped "conventions" so that "them" would refer to the decorators, not the convention. Your wording works though!